src/main/java/org/assertj/core/api/AbstractClassAssert.java

```diff
@@ -647,4 +647,59 @@ public SELF hasPublicMethods(String... methodNames) {
     classes.assertHasPublicMethods(info, actual, methodNames);
     return myself;
   }
+
+  /**
+   * Verifies that the actual {@code Class} has the given package name.
```

Comment:
As we are still java 8 based and `Class#getPackageName()` comes from java 9+, I'd explicitly mention the relationship with `Class#getPackage()` behavior:

```suggestion
   * Verifies that the actual {@code Class} has the given package name (as in {@link Class#getPackage()}).
   *
```

Reply:
Fine, I thought that references to inner logic should be avoided in API docs. But if you think it help understand the assertion let's add it. Should it be "{@link Class#getPackage().getName()}" anyway?